### PR TITLE
VIA Support: KBD75 rev1/rev2

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

Adding VIA support for rev1/rev2 of the KBD75. 

The rev2 is a very similar PCB but it has one major change, in that it allows the numpad layout. Prior to this change, the VID and PID would be the same leading to VIA users being able to set a numpad layout without the PCB actually having that capability. This will confuse people. I've added `config.h` files with PIDs to each rev directory to help mitigate this. 

Changes on VIA and on this firmware were tested by KlondikeBahr as I do not own any kbd75. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mechmerlin** [comment]
Build failure on yan's keymap =P

File: keyboards/kbdfans/kbd75/rev1/config.h [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#define PRODUCT_ID      0x6060
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75/rev1/config.h]:
```suggestion
#pragma once

#define PRODUCT_ID      0x6060
```

File: keyboards/kbdfans/kbd75/rev2/config.h [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#define PRODUCT_ID      0x6061
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75/rev2/config.h]:
```suggestion
#pragma once

#define PRODUCT_ID      0x6061
```

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**